src/kOS.Safe/Compilation/KS/LineCol.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace kOS.Safe.Compilation.KS
+{
+    /// <summary>
+    /// Just a dumb simple tuple of line and column, to be returned in
+    /// places where the compiler needs to treat the pair of them like
+    /// a single return value from a method.
+    /// </summary>
+    public class LineCol
+    {
+        public short Line { get; set; }
```

Comment:
doesn't really matter.  There's currently no use of their mutability so it could be changed.